addons/purchase/models/purchase.py

```diff
@@ -398,35 +399,132 @@ def _add_supplier_to_product(self):
                 except AccessError:  # no write access rights -> just ignore
                     break
 
-    def action_view_invoice(self):
+    def action_create_invoice(self, grouped=False):
+        """
+        Create the invoice associated to the SO.
```

Comment:
Line break after """ is not a good practice. Check https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html